cmd/gossamer/configcmd.go

```diff
@@ -52,7 +53,7 @@ func makeNode(ctx *cli.Context) (*dot.Node, *dot.Config, error) {
 		return nil, nil, err
 	}
 
-	log.Info("🕸\t Configuring node...", "datadir", currentConfig.Global.DataDir, "protocol", currentConfig.Network.ProtocolID, "bootnodes", currentConfig.Network.Bootnodes)
+	log.Info("[gossamer] Initializing node...", "datadir", currentConfig.Global.DataDir)
```

Comment:
no more protocol and bootnodes? lol

Reply:
it felt like too much information, I actually wanted to add more initially then realized the most we just need to know which initialized node we are running and trace will provide more information about updated configuration values that differ from the defaults

Reply:
okay, as long as it's somewhere, `Debug` might be nice